### ISSUE TITLE
Plugin appears to be abandoned ... 

### ISSUE THREAD

**Issue by catafest** [aramk/crayon-syntax-highlighter#485]
This is true?
_Details: It was last updated 5 years 11 months ago and tested up to WordPress 4.2.0. Plugins can be removed from wordpress.org for various reasons. This can include benign issues like a plugin author discontinuing development or moving the plugin distribution to their own site, but some might also be due to security issues. In any case, future updates may or may not be available, so it is worth investigating the cause and deciding whether to temporarily or permanently replace or remove the plugin_

**Comment by urvanov-ru** [comment]
Yes, the author abandoned it. But I am developing a forked copy Urvanov Syntax Highlighter. It is almost the same but with Gutenberg Support and with fixes.

**Comment by urvanov-ru** [comment]
As an another option you can switch to EnlighterJS. I know that it has support for crayon tags.
